src/components/Shared/FooterInfo.js

```diff
@@ -52,10 +59,8 @@ const FooterInfoCard = function FooterInfoCard({ isDrPage = false }) {
       >
         GURS
       </a>
-      {/*
       <br />
-      {t('footer.lastChange')}: <strong>{lastChange()}</strong>
-      */}
+      {t('footer.lastChange')}: <strong>{lastChange(drTs)}</strong>
```

Comment:
Could this be changed to use i18next interpolation for better flexibility in translations?
```suggestion
      {t('footer.lastChange', { date: new Date(drTs) })}
```

or remove the placeholders from translations: https://github.com/sledilnik/zdravniki/blob/947357458bafa53846c299c288b2d89173458ada/src/locales/en.json#L79

Reply:
sure. I did't know it's possible.

Reply:
Done. Slightly different as you suggested.